### PR TITLE
feat(global) adding support with docker

### PR DESCRIPTION
**Pull request by mario21ic**
Adding support Docker with PHP v5.6 and MySQL 5.5, also added the script start.sh to initialize the project

### CODE REVIEW

File: config/config.inc.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #   WARNING: The database specified under db_database WILL BE ENTIRELY DELETED during setup.
 #   Please use a database dedicated to DVWA.
 $_DVWA = array();
-$_DVWA[ 'db_server' ]   = '127.0.0.1';
+$_DVWA[ 'db_server' ]   = 'db';
```
Comment (ethicalhack3r) [inline config/config.inc.php]: I would leave this as 127.0.0.1

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    #links:
+    #  - db
+    ports:
+      - 80:80
+
+  db:
+    image: mysql:5.5
+    environment:
+      "MYSQL_DATABASE": "dvwa"
+      "MYSQL_ROOT_PASSWORD": "p@ssw0rd"
```
Comment (ethicalhack3r) [inline docker-compose.yml]: Can these be read from the config file directly? Would save some debugging for novice users if the configs here do not match the ones in config.php

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+chmod 775 -R ./hackable/uploads/
+chmod 775 ./external/phpids/0.6/lib/IDS/tmp/phpids_log.txt
+docker-compose up -d
```
Comment (ethicalhack3r) [inline start.sh]: Personally I wouldn't include this file

File: docker/php.ini [hunk 4]
```diff
@@ -0,0 +1,2030 @@
...
+; should be disabled, as enabling it may result in issues when generating XML
+; documents, however this remains supported for backward compatibility reasons.
+; Note that this directive does not control the <?= shorthand tag, which can be
+; used regardless of this directive.
+; Default Value: On
+; Development Value: Off
+; Production Value: Off
+; http://php.net/short-open-tag
+short_open_tag = Off
+
```
Comment (ethicalhack3r) [inline docker/php.ini]: I'd probably enable this as when enabled it is less secure

File: config/config.inc.php (hunk 1 above)
Comment (mario21ic) [inline config/config.inc.php]: Hi, ok, I leave this as 127.0.0.1

File: docker-compose.yml (hunk 2 above)
Comment (mario21ic) [inline docker-compose.yml]: Well, docker compose cannot read from php file, but yes from a env file (key value) https://docs.docker.com/compose/compose-file/#envfile

File: start.sh (hunk 3 above)
Comment (mario21ic) [inline start.sh]: I will change the name to start-docker.sh, maybe?

File: docker/php.ini (hunk 4 above)
Comment (mario21ic) [inline docker/php.ini]: I forget it, now I'll fix it

**Comment by mike-bailey** [comment]
I'm closing this because I'm already being bent over a barrel to support standard Docker, which I'm about to, and this may be out of date now, but thank you for the help. @ethicalhack3r can of course overrule me.

**Pull request closed without merging**